services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -366,6 +366,7 @@ export const DocumentTable: FC<DocumentTableProps> = ({
         setCompressionModalVisible={setCompressionModal}
         isCompressionModalVisible={isCompressionModal}
         compressionInProgress={setCompressionInProgress}
+        showArchiveDownloadWarning={true}
```

Comment:
would true be a reasonable default for this prop?

Reply:
@taraepp that is a good point. I initially thought to set this prop as a way to temporarily disable the archive warning modal that pops up because NoW section hasn't had archive and versioning setup yet. At the same time, it is possible for other sections not just Major Projects or NoW to use document compression within the DocumentTable since DocumentTable is used in alot of different areas that also don't have archive or versioning files.

Would it make more sense to do it this way where we check the selected rows to see if any selection contains property `is_archived === true || is_latest_version === false`, and if so then we set the boolean for `showArchiveDownloadWarning ` to true, else false

Reply:
Actually scratch what I mentioned above, the user is not able to select any archived documents or older versions of documents when they bulk download. I think just using DocumentTable's props `showVersionHistory `and `canArchiveDocuments` can work. So something like `showDownloadWarning={showVersionHistory || canArchiveDocuments}`

Reply:
Adjusted the DocumentTable's showDownloadWarning default value to be based on `showVersionHistory || canArchiveDocuments`